standard/lexical-structure.md

```diff
@@ -756,18 +756,18 @@ The value of a real literal of type `float` or `double` is determined by using t
 > *Example*:
 >
 > ```csharp
-> 1.234_567              // double
-> .3e5f                  // float
-> 2_345E-2_0             // double
-> 15D                    // double
-> 19.73M                 // decimal
-> 1.F                    // invalid; ill-formed (parsed as "1." and "F")
-> _1.2F                  // invalid; no leading _ allowed
-> 1_.2F                  // invalid; no trailing _ allowed in integer part
-> 1._234                 // invalid; no leading _ allowed in fraction
-> 1.234_                 // invalid; no trailing _ allowed in fraction
-> .3e_5F                 // invalid; no leading _ allowed in exponent
-> .3e5_F                 // invalid; no trailing _ allowed in exponent
+> 1.234_567      // double
+> .3e5f          // float
+> 2_345E-2_0     // double
+> 15D            // double
+> 19.73M         // decimal
+> 1.F            // parsed as a member access of F due to non-digit after .
+> _1.2F          // invalid; _1 is an identifier due to leading _
```

Comment:
This is the identifier `_1` and the literal `.2F` – lexically correct, syntactically incorrect... I'd probably drop this example!

Reply:
Given that we have other examples with leading underscores, that probably makes sense. Will leave it for this minute to let others comment, but with an intention to drop it.